.github/workflows/publish-docs.yml

```diff
@@ -29,3 +29,42 @@ jobs:
 
       - timeout-minutes: 25
         run: earthly-ci --no-output ./docs/+deploy-prod --NETLIFY_AUTH_TOKEN=${{ secrets.NETLIFY_AUTH_TOKEN }} --NETLIFY_SITE_ID=${{ secrets.NETLIFY_SITE_ID }} --COMMIT_TAG=${{ inputs.tag }}
+        
+  pdf:
+    needs: setup
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AZTEC_BOT_GITHUB_TOKEN }}
+          path: aztec-packages
+      - name: Checkout tools repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AZTEC_BOT_GITHUB_TOKEN }}
+          repository: AztecProtocol/protocol-specs-pdf
+          path: protocol-specs-pdf
+      - name: Install Prince
+        run: |
+          curl https://www.princexml.com/download/prince-14.2-linux-generic-x86_64.tar.gz -O
+          tar zxf prince-14.2-linux-generic-x86_64.tar.gz
+          cd prince-14.2-linux-generic-x86_64
+          yes "" | sudo ./install.sh
+      - name: generate pdf
+        run: |
+          cd aztec-packages/docs
+          yarn serve
```

Comment:
I believe the docs are built earlier, how can I access the build so I don't need to build them again and can just serve them? Will it work to serve them locally in Earthly, or should I try to grab the pages from the published url above?